sdk/cosmos/azure-cosmos/samples/test_xq.py

```diff
@@ -0,0 +1,82 @@
+import time
```

Comment:
why have we added this test in samples ?
Also, I see license header is missing for this one and some other samples, we should add it.